OrbitGl/App.cpp

```diff
@@ -873,6 +873,7 @@ void OrbitApp::OnProcessSelected(int32_t pid) {
   std::shared_ptr<Process> process = FindProcessByPid(pid);
   CHECK(process != nullptr);
   Capture::SetTargetProcess(std::move(process));
```

Comment:
Should we set taget process after modules have successfully been updated?

Reply:
I guess UpdateModules uses the process. But we can pass it as a param. Maybe this is better. I'll look into this again.

Reply:
UpdateModuleList(pid) also uses FindProcessById(pid). You can probably leave it as is. If I understood the problem, the problem is having a delay between SetTargetProcess and FireRefreshCallbacks.

Reply:
Yep you were right. I implemented it this way. Seems to be the proper solution for the issue.